.github/workflows/snap-deployment.yml

```diff
@@ -1,6 +1,6 @@
-name: Deployment
```

Comment:
Why did you changed that?

Reply:
I confused it, I though it's a separate file.